xyz-models/src/main/java/com/here/xyz/models/hub/Space.java

```diff
@@ -317,13 +325,31 @@ public Extension getExtension() {
 
   public void setExtension(final Extension extension) {
     this.extension = extension;
+    if (extension != null) {
+      setExtendsFrom(extension.getSpaceId());
+    }
   }
 
   public Space withExtension(final Extension extension) {
     setExtension(extension);
     return this;
   }
 
+  public String getExtendsFrom() {
```

Comment:
No changes on the Space model are necessary for the changes in the (only dynamo related) storage representation. As the new (additional field) extendsFrom is only needed for technical reasons only by the DynamoSpaceConfigClient

Reply:
ok